README.md

```diff
@@ -0,0 +1,24 @@
+<h3 align="center">vacay quiz</h3>
+<h4 align="center"><i>Learning Made Fun</i></h4>
+<p align="center"><a href="https://vacay-quiz.netlify.app">Play quizzes »</a></p>
+
+## Overview
+Vacay quiz is a web application for playing quizzes based on various Indian destinations. Vacay quiz is publicly hosted on Netlify at https://vacay-quiz.netlify.app/.
+
+## Demo
+![image](https://user-images.githubusercontent.com/56336326/154799966-7213f9cc-d502-4b3b-8387-a7e8de89af89.png)
+
+## Features
+- Home page - https://vacay-store.netlify.app/
+- Quizzes by Category - https://vacay-quiz.netlify.app/category.html
+- Rules Page for while attempting Quiz - https://vacay-quiz.netlify.app/rule.html
+- MCQ based Quiz Question Page - https://vacay-quiz.netlify.app/quiz.html
+- Result Page with Answers & Score - https://vacay-quiz.netlify.app/result.html
+
+## Tech stack
+* HTML
+* CSS
+
+## Socials
+* https://twitter.com/_anik_31
```

Comment:
Changes - It would look great if your links are clickable texts. For your reference, [this is google](https://www.google.com/)

Check this out -
https://www.markdownguide.org/basic-syntax/#links

Reply:
done